screens/NativeModule.tsx

```diff
@@ -7,13 +7,22 @@ import { Camera } from 'expo-camera';
 import CameraComponent from '../components/CameraComponent';
 import MorseForm from '../components/MorseForm';
 import { translateToMorse } from '../morseMessenger/morseFunctions';
+import { dictOptions } from '../morseMessenger/morseDictionary';
 
 const TORCH_ON = 'torch';
-const TORCH_OFF = '';
+const TORCH_OFF = 'off';
+const PAUSE = 1000;
+const FLASH_PAUSE = 400;
+const FLASH_SHORT = 100;
+const FLASH_LONG = 700;
+const s = dictOptions.s;
```

Comment:
IMO there's no need for these consts, it's better to just use `dictOptions` directly (it's not that long and it's just three if-s)